queries/muttrc/highlights.scm

```diff
@@ -20,10 +20,13 @@
 
 (regex) @string.regexp
 
-[
-  (option)
-  (command_line_option)
-] @variable.builtin
+(command_line_option) @variable.builtin
+
+((option) @variable.builtin
+  (#not-lua-match? @variable.builtin "^my_"))
+
+((option) @variable
+  (#lua-match? @variable "^my_"))
```

Comment:
having both be a lua match is redundant, pick a base case and a more specific case to follow below

Reply:
Fixed.